sphinxcontrib/openapi.py

```diff
@@ -122,10 +122,30 @@ def _httpresource(endpoint, method, properties):
     yield ''
 
 
-def openapi2httpdomain(spec):
+def _clean_paths_option(value):
```

Comment:
I went with `string_newline_list` to convey the extra detail that the string is split by newlines rather than something else.